.github/workflows/publish-image.yaml

```diff
@@ -1,29 +1,18 @@
 name: publish image
-
-on:
-  push:
-    branches:
-      - master
-    paths:
-      - 'package.json'
-
+on: push
```

Comment:
Can we run this job only on pushes to a release branch, e.g. `release/1.2.3` ?

Reply:
Done :+1: we might want to consider building on PRs as well but I'm not sure at this point.